sdk/core/README.md

```diff
@@ -37,6 +38,10 @@
     - [Validate Holder Response on Offer Request](#validate-holder-response-on-offer-request)
     - [Sign multiple credentials](#sign-multiple-credentials)
     - [Generate signed credential](#generate-signed-credential)
+  - [Revocation](#revocation)
+    - [Flow](#Revocation Flow)
+    - [Issuance](#issuance of Revocable credential)
+    - [Revocation](#Revocation of Revocable credential)
```

Comment:
@Volland looks like this format is incorrect. I think you should have a `*` before [Revocation] 
Right now it looks like this:
![image](https://user-images.githubusercontent.com/3696612/117291957-3eabcb80-ae8d-11eb-9fa3-13d2cdf7e081.png)